src/graphql/resolveConnection.js

```diff
@@ -8,6 +8,7 @@ const resolveConnection = (
   getFromClause = constant(table.getIdentifier()),
 ) => {
   const columns = table.getColumns()
+  const fullPrimaryKey = table.getPrimaryKeys()
```

Comment:
I am not sure what you mean with **full**PrimaryKey. Though I am guessing it just means it has all the primary keys including a pk that is used to order the collection — which you later filter out. Why not just call this `primaryKeys`?